service/sg-windows/build.cmd

```diff
@@ -1,4 +1,7 @@
 @echo off
+
+SET GOPATH=%cd%
```

Comment:
Please store the existing GOPATH in a temp var and restore after build finishes.